cdap-api/src/main/java/co/cask/cdap/api/flow/Flow.java

```diff
@@ -36,7 +36,9 @@
   /**
    * Configure the {@link Flow} by returning a {@link FlowSpecification}.
    *
-   * @return An instance of {@link FlowSpecification}.
+   * @return An instance of {@link FlowSpecification}
+   * @deprecated Use {@link AbstractFlow#configureFlow} instead
    */
+  @Deprecated
```

Comment:
shouldn't there be a new configure method? Otherwise this class just has a single deprecated method